src/main/java/com/epam/jmp/redislab/api/validator/RequestValidator.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.jmp.redislab.api.validator;
+
+import com.epam.jmp.redislab.api.RateLimitRequest;
+import com.epam.jmp.redislab.api.RequestDescriptor;
+
+public class RequestValidator {
```

Comment:
Looks like here should be `@Component`

Reply:
thanks!

Reply:
![image](https://github.com/ShiniTada/rate-limit-business-service/assets/41171038/a89982ee-1c07-42fa-8756-066eaec8de2c)
Now it is working